src/api/teams/controllers/create-team.js

```diff
@@ -40,19 +40,19 @@ const createTeamController = {
       if (!gitHubExists) {
         throw Boom.badData('Team does not exist in GitHub')
       }
-    }
-
-    try {
-      const team = await createTeam(request.msGraph, request.db, dbTeam)
 
       // This is still experimental, so we're just going to log the error from this bit for now.
       try {
-        await updateSharedRepoAccess(request.octokit, payload.github)
+        await addSharedRepoAccess(request.octokit, payload.github)
```

Comment:
This has been moved as we weren't looking to see whether the GitHub details weren't null ( I believe this was just silently failing in these cases)